ebos/collecttoiorank.hh

```diff
@@ -207,11 +207,11 @@ public:
     };
 
     /// \brief Communication handle to scatter the global index
-    template<class Mapper>
+    template<class EquilMapper, class Mapper>
```

Comment:
I am a little confused about the name EquilMapper, would not SendMapper and RecvMapper be better?

Reply:
Naming is consistent with previous naming of the variables used further [down](/OPM/opm-simulators/blob/608b317ecccf49ac9c8d27847332f2709ace8cd0/ebos/collecttoiorank.hh#L322-L345) (grid, equilGrid. Equil indicates that it is for the global grid.

But I agree using send and receive in the constructor is not. What should we do?

Reply:
Thanks for your comment. I think "equil" meaning the full, global grid is not very intuitive (unless you have been reading too much of the OPM code...), perhaps we should use the term "global" instead? (Although that means "all logical cartesian cells" in some contexts, I guess we should replace them with "logical cartesian" as well.)

Reply:
Since renaming allocurrences would change other parts of the file, should I do that here or in another PR?

Reply:
Another PR is fine. I'll run jenkins and merge as is if green.